client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/style.scss

```diff
@@ -47,12 +47,19 @@
 
 					.item-preview__header-summary-link {
 						display: flex;
+						gap: 2px;
 						align-items: center;
-						font-size: rem(18px);
+						font-size: rem(16px);
 						font-weight: 400;
 						color: var(--color-accent-70);
-						line-height: 26px;
+						line-height: 24px;
```

Comment:
I don't advise using a pixel unit when setting line-height, as it does not adjust relative to the font size. We can use 1.5 here, which is equivalent to 24px (24/16).

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/90435/commits/b527c76a96bc6edd4b413a99e3d22630eb3ff068 but noting there are a bunch of line-heights defined in pixels besides this one.